include/mbedtls/config.h

```diff
@@ -3617,7 +3617,7 @@
 
 /* ECP options */
 //#define MBEDTLS_ECP_MAX_BITS             521 /**< Maximum bit size of groups */
-//#define MBEDTLS_ECP_WINDOW_SIZE            6 /**< Maximum window size used */
+//#define MBEDTLS_ECP_WINDOW_SIZE            4 /**< Maximum window size used */
```

Comment:
Technically yes, this is a number of bits, but knowing that doesn't help at all in selecting a value. There's some documentation on that [in `ecp.h`](https://github.com/ARMmbed/mbedtls/blob/mbedtls-2.26.0/include/mbedtls/ecp.h#L259), but it isn't in Doxygen. Improving the Doxygen rendering is out of scope here.

Reply:
I think it's quite irrelevant that it's a number of bits, but what's relevant is that the cost of computing a multiplication goes `A * 2^w + B / w` for some constants A and B, where `w` is the selected window size, while the memory cost goes `C * 2^w` for another constant C. Perhaps instead of calling it "window size" (which indeed begs the question of the unit) we should have called it "RAM/speed trade-off level"...

Reply:
I understand it does not help in selecting a value but it clarifies what it is. The size is not in bytes or kB or MB, it is a number of bits. This is not at all a blocker for me but what would it be a problem to change the comment to "Maximum window size used in number of bits.". This is just more precise to me.

Reply:
This is preexisting so I'm not going to work on it here. Filed as https://github.com/ARMmbed/mbedtls/issues/4700.

I disagree with mentioning that it's a number of bits. That's not more precise, it's just misleading.

Reply:
That's maybe just my background in physics in play here but to me a size without a unit is a problem. Maybe this is not a size then I don't know. Anyway closing this conversation as it is not a big deal, also not sure it is worth an issue.